dolphinscheduler-dist/release-docs/LICENSE

```diff
@@ -584,3 +584,64 @@ Apache 2.0 licenses
 BSD licenses
 ========================================
     d3 3.5.17: https://github.com/d3/d3 BSD-3-Clause
+
+
+========================================================================
+UI-Next related licenses
+========================================================================
+
+The following components are used in UI-Next.See project link for details.
+The text of each license is also included at licenses/ui-next-licenses/LICENSE-[project].txt.
+
+========================================
```

Comment:
Seems you miss a section for those fonts `SIL OPEN FONT LICENSE`?

Reply:
I did ignore the font problem, but the font does not have a corresponding version number, nor does it seem to have a corresponding github repository address or npm address. How to deal with this.

Reply:
FiraCode has a repo and version: https://github.com/tonsky/FiraCode
https://github.com/mozilla/Fira

Version numbers are not a must in this file, but this file serves as a "Table of Content" for the licenses, "The text of each license is also included at licenses/ui-next-licenses/LICENSE-[project].txt.", if the licenses are not listed in this file, users might ignore those licenses and don't check files under `licenses/.........`

Reply:
OK, I'll deal with it now.